crates/kornia-imgproc/src/features.rs

```diff
@@ -82,6 +91,183 @@ pub fn hessian_response(src: &Image<f32, 1>, dst: &mut Image<f32, 1>) -> Result<
     Ok(())
 }
 
+/// A builder object to initialize Harris response
+pub struct HarrisResponse {
+    image_size: ImageSize,
+    k: f32,
+    grads_mode: GradsMode, // TODO
```

Comment:
can we just remove it for now untill we don't implement it. Thows some warnings, same for sigmas